test/integration/uninstall/spec.rb

```diff
@@ -0,0 +1,27 @@
+control 'basic_uninstall' do
+  impact 0.4
+  title 'Uninstall Splunk'
+  tag 'splunk_install'
+
+  describe package(os.windows? ? 'Splunk Enterprise' : 'splunk') do
+    it { is_expected.not_to be_installed }
+  end
+
+  describe command('ps -eo comm= | grep splunk') do
+    its('exit_status') { should eq 1 }
```

Comment:
Just wondering, why the difference between install and uninstall for `is_expected.to eq 1` and `should eq 1`?